lib/heartcheck/controllers/inspect.rb

```diff
@@ -0,0 +1,30 @@
+module Heartcheck
+  module Controllers
+    class Inspect < Base
+      def index
+        results = {
+          application_name: application_name,
+          environment: environment,
+          dependencies: []
+        }
+
+        checks = Heartcheck.checks
+        results[:dependencies] += checks.reduce([]) do |acc, elem|
+          acc << elem.uri_info
+        end.flatten
+
+        MultiJson.dump(results)
+      end
+
+      private
+
+      def application_name
+        ENV.fetch('HEARTCHECK_APP_NAME')
+      end
+
+      def environment
+        ENV.fetch('RAILS_ENV')
```

Comment:
`RAILS_ENV` is not available in non-rails applications.

Also, in our apps, we prefer to use `RAILS_ENV=production` in QA because the behavior of the app may change according to this env var (some gems can use `Rails.env.production?`) and we want to have QA as similar as possible to production. We prefer `RAILS_ENV=production` in QA, also because according to [12FactorApp recommendation](https://12factor.net/config) to have different environments for "qa/staging/sysint" is not scalable.

I think we should use envs prefixed by `HEARTCHECK_` here, in this case `HEARTCHECK_ENV`, what do you think? :innocent: